src/org/ods/orchestration/usecase/BitbucketTraceabilityUseCase.groovy

```diff
@@ -0,0 +1,221 @@
+package org.ods.orchestration.usecase
+
+import org.ods.orchestration.util.Project
+import org.ods.services.BitbucketService
+import org.ods.util.IPipelineSteps
+
+import java.text.SimpleDateFormat
+
+@groovy.lang.Grab('com.xlson.groovycsv:groovycsv:1.3')
+import com.xlson.groovycsv.CsvParser
+
+class BitbucketTraceabilityUseCase {
+
+    private static final String CSV_FILE = "source-code-review.csv"
+    private static final int PAGE_LIMIT = 10
+
+    private final BitbucketService bitbucketService
+    private final IPipelineSteps steps
+    private final Project project
+
+    BitbucketTraceabilityUseCase(BitbucketService bitbucketService, IPipelineSteps steps, Project project) {
+        this.steps = steps
+        this.project = project
+        this.bitbucketService = bitbucketService
+    }
+
+    /**
+     * Create a CSV file that contains the following records
+     * for every merge event into the integration branch of every ODS component:
+     * @return absolutePath of the created file
+     */
+    @SuppressWarnings(['JavaIoPackageAccess'])
+    String generateSourceCodeReviewFile() {
+        def file = new File("${steps.env.WORKSPACE}/${CSV_FILE}")
```

Comment:
We typically use some namespace directories to differentiate different types of files. Please avoid writing this file directly into env.WORKSPACE. See MROPipelineUtil class members for examples.

Reply:
Created in new "review" folder